### ISSUE TITLE
Integrations are missing an Exists function

### ISSUE THREAD

**Issue by cory-signalfx** [splunk-terraform/terraform-provider-signalfx#44]
Oops :(

**Comment by asmod3us** [comment]
Is this scenario possibly related? If I delete an integration via SignalFx Web UI and attempt a `terraform refresh` I get a 404 error and the refresh aborts. Same for `terraform destroy`.

```
{
  "code" : 404,
  "message" : "Unable to find the given integration."
}
```

**Comment by a-staebler** [comment]
Is this scenario possibly related? If I delete an integration via SignalFx Web UI and attempt a `terraform refresh` I get a 404 error and the refresh aborts. Same for `terraform destroy`.

```
{
  "code" : 404,
  "message" : "Unable to find the given integration."
}
```

**Comment by cory-signalfx** [comment]
Yes! I'll get a fix for this in today.

**Comment by cory-signalfx** [comment]
Fixed in #50, #51, and #52
